pages/index.js

```diff
@@ -1,19 +1,34 @@
-import React from "react";
+import React, { useState } from "react";
 import fs from "fs";
 import xml2js from "xml2js";
+import { Wrapper, Uber, Orderli, Lila } from "./styles";
 
 const Playlist = ({ top100 }) => {
+  const [selectedFile, setSelectedFile] = useState(null);
+
+  const handleFileUpload = (event) => {
+    setSelectedFile(event.target.files[0]);
+  };
+
   return (
-    <div>
-      <h1>Top 100 meistgespielten Songs</h1>
-      <ol>
+    <Wrapper>
+      <Uber>Top 100 meistgespielten Songs</Uber>
+      <Orderli>
         {top100.map((song) => (
-          <li key={song.id}>
-            {song.title} von {song.artist} ({song.playCount} Mal gespielt)
-          </li>
+          <Lila key={song.id}>
+            <span style={{ color: "hotpink" }}>{song.title}</span>
```

Comment:
inline styling is not the best solution, you should use styledComponents here as well